pkgs/tools/networking/mpssh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, openssh }:
+
+stdenv.mkDerivation rec {
+  pname = "mpssh";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    sha256 = "0iibzb9qrwf9wsmp2b8j76qw66ql8fw5jx02x82a22fsaznb5dng";
+    rev = "${version}";
+    repo = pname;
+    owner = "ndenev";
+  };
+
+  buildInputs = [ openssh ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace /usr/local $out
```

Comment:
You can use `"BIN=${placeholder "out"}/bin"` in `makeFlags`.

https://github.com/ndenev/mpssh/blob/6c577ee51e9ec7196c8ddc34768baeba317af8b5/Makefile#L8